src/module/apps/migration-summary.ts

```diff
@@ -66,39 +66,47 @@ export class MigrationSummary extends Application<MigrationSummaryOptions> {
 
     override activateListeners($html: JQuery): void {
         super.activateListeners($html);
+        const html = $html[0];
 
-        $html.find("button[data-action=remigrate]").on("click", async (event) => {
-            const { LATEST_SCHEMA_VERSION, RECOMMENDED_SAFE_VERSION } = MigrationRunner;
-            const lowestVersions = {
-                actor:
-                    game.actors.size > 0
-                        ? Math.min(...game.actors.map((a) => a.schemaVersion ?? 0))
-                        : LATEST_SCHEMA_VERSION,
-                item:
-                    game.items.size > 0
-                        ? Math.min(...game.items.map((a) => a.schemaVersion ?? 0))
-                        : LATEST_SCHEMA_VERSION,
-            };
-            const lowestSchemaVersion = Math.max(
-                Math.min(lowestVersions.actor, lowestVersions.item),
-                RECOMMENDED_SAFE_VERSION
-            );
+        for (const button of html.querySelectorAll("button")) {
```

Comment:
Wouldn't you be able to avoid a layer of indentation if you query for `button[data-action=remigrate]`? What's the performance difference in doing it this way?